README.md

```diff
@@ -0,0 +1,46 @@
+# Pawntastic
+An object-oriented game written in Java using libGDX.
+
+<!-- Badge showing test suite status -->
+[![Test Suite](https://github.com/martinjonsson01/OOPP-WITH-THE-BOIS/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/martinjonsson01/OOPP-WITH-THE-BOIS/actions/workflows/test.yml)
+
+## Understanding the Architecture
+There is a UML class diagram of each package located in the README file of each package directory. It is visible directly when navigating to the package directory on GitHub, but if viewing the files locally the raw images can be found in `documents/diagrams/`.
+
+![com.thebois package class diagram](documents/diagrams/com.thebois.jpg "com.thebois package class diagram")
+
+## Running the Game
+
+### Prerequisites
+
+* Java 15 or later
+
+To run the latest version of the game, download the jar file from [releases](https://github.com/martinjonsson01/OOPP-WITH-THE-BOIS/releases/) and run it using the command:
+```terminal
+java -jar ColonyManagement-vX.X.X.jar
+```
```

Comment:
Just downloading java 16 does not work.

After that I tried/tested installing JRE, then I got a JNI error and  a exception "java.langUnsupportedClassVersionError"

And set both path and JAVA_HOME variables to 16.

Wouldn't start on my vm since it seems to require drivers for libgdx that isnt available in vm (not straight out of the box at least). Got the same error with java 15.

But worked fine on my machine outside the vm.